tests/unit/components/test_pebble_component.py

```diff
@@ -13,13 +13,30 @@
 from ops import ActiveStatus, WaitingStatus
 
 import charmed_kubeflow_chisme.components.pebble_component
-from charmed_kubeflow_chisme.components import ContainerFileTemplate, LazyContainerFileTemplate
+from charmed_kubeflow_chisme.components import (
+    ContainerFileTemplate,
+    LazyContainerFileTemplate,
+    get_event_from_charm,
+)
 
 
 class TestPebbleComponent:
     name = "test-component"
     container_name = "test-container"
 
+    def test_get_event_from_charm(self, harness_with_container):
+        """Test that get_event_from_charm returns the events requested."""
+        event_type = "mock_event"
+        expected_event_name = "test_container_mock_event"
+        mock_event_value = "MockEvent"
+
+        setattr(harness_with_container.charm.on, expected_event_name, mock_event_value)
```

Comment:
I had in mind to test with the actual events without doing `setattr`, I expected the harness charm to have `pebble_ready` and `pebble_check_failed` etc once it is started. Can you confirm if this is possible?